packages/graphback-cli/src/templates/configTemplates.ts

```diff
@@ -70,21 +62,60 @@ export const askForClient = async (): Promise<boolean> => {
  * Create config file with db info
  */
 export const createConfig = async (database: string, client: boolean) => {
-  const configPath = `${process.cwd()}/graphback.json`
+  const configPath = `${process.cwd()}/.graphqlrc.yml`
 
   const dockerComposePath = `${process.cwd()}/docker-compose.yml`
-  const config = {}
   const [dbConfig, dockerCompose] = getConfig(database)
-  config['db'] = {
-    dbConfig: JSON.parse(dbConfig),
-    database: database
+  const graphqlConfig: IGraphQLConfig = {
+    schema: './src/schema/*.gql',
```

Comment:
`documents` config falls under `./client` but `schema` config does not fall under `server` (as it does in template config. For this reason `graphback config` does not work out of the box.

Reply:
Graphback config will not work out of the box as we never know what folder structure people will have. Adjusting it to our example app makes no sense at all.

Reply:
Do you have any proposition for this? I came up with some assumption that we will be running this in the server side root and client is just optional

Reply:
Ah okay that's fair enough then.